crates/astria-celestia-client/src/client.rs

```diff
@@ -260,6 +257,8 @@ fn assemble_blobs_from_sequencer_block_data(
         chain_ids.push(chain_id);
     }
 
+    let sequencer_namespace =
+        celestia_namespace_v0_from_hashed_bytes(header.chain_id.clone().as_bytes());
```

Comment:
should this be Namespace::const_v0 - i think from_hashed_bytes are for if the input is already hashed?